diagnostic.md

```diff
@@ -36,4 +41,8 @@ Record your responses inside the fenced code blocks below each question.
 
     ```html
     <!-- your response here -->
+    {{#each my-phone.phoneNum as |phoneNum|}}
+      {{contacts/my-contact/my-phone phoneNum=phoneNum}}
+    {{/each}}
```

Comment:
you don't need the `contacts/my-contact/`.  we want it to look something like this:
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```